src/tabs/__integ__/tabs.test.ts

```diff
@@ -58,8 +58,7 @@ class TabsPage extends BasePageObject {
 const setupTest = (testFn: (page: TabsPage) => Promise<void>, smallViewport = false) => {
   return useBrowser(async browser => {
     const page = new TabsPage(browser);
-    // TODO: Remove VR flag once AWSUI-18703 is fixed
-    await browser.url('#/light/tabs/responsive-integ/?motionDisabled=true&visualRefresh=false');
+    await browser.url('#/light/tabs/responsive-integ');
```

Comment:
The issue is fixed and we can go to the defaults now. Needed to update one test below to make it work in visual refresh spacing and font sizes